src/wp-includes/class-wp-block-list.php

```diff
@@ -0,0 +1,188 @@
+<?php
+/**
+ * Blocks API: WP_Block_List class
+ *
+ * @package WordPress
+ * @since 5.5.0
+ */
+
+/**
+ * Class representing a list of block instances.
+ */
+class WP_Block_List implements Iterator, ArrayAccess, Countable {
+
+	/**
+	 * Original array of parsed block data.
+	 *
+	 * @var array|WP_Block[]
```

Comment:
The `array` part of the type isn't necessary.

Reply:
I think it's correct as is, see line 43:

`* @param array|WP_Block[]       $blocks            Array of parsed block data, or block instances.`

Reply:
Ah maybe we can include that in the property description? Also perhaps expand to be `array[]|WP_Block[]`? 

To me it reads at first glance of saying this is an array and its an array of `WP_Block` instances. I think those changes might make that more clear?

Reply:
Good call, I checked the code (I didn't author it) and it looks like what you proposed makes the most sense 👍 Updated.